### ISSUE TITLE
No spaced after `-` in `BSD-3-Clause-Sun`

### ISSUE THREAD

**Issue by sschuberth** [nexB/scancode-toolkit#3665]
https://github.com/nexB/scancode-toolkit/blob/9d466d3de337e6cc4487fa7a5c933a02d66d9c39/src/licensedcode/data/licenses/bsd-3-clause-sun.LICENSE#L19

was noticed over [here](https://github.com/oss-review-toolkit/ort/pull/8308#discussion_r1494687433).

**Comment by pombredanne** [comment]
Thanks!
Note that this has been this way upstream such as in https://github.com/antuansoft/jdk1.5.0_13/blob/3d72a1ad826fe806c6fe162ccd26743e4fcbad8e/demo/jfc/Metalworks/src/MetalThemeMenu.java#L9

**Comment by sschuberth** [comment]
> Note that this has been this way upstream

Ok, but the referenced https://github.com/xmlark/msv/blob/b9316e2f2270bc1606952ea4939ec87fbba157f3/xsdlib/src/main/java/com/sun/msv/datatype/regexp/InternalImpl.java has it different, though. I guess it would make sense to align the text with the reference either way?

**Comment by pombredanne** [comment]
> Ok, but the referenced https://github.com/xmlark/msv/blob/b9316e2f2270bc1606952ea4939ec87fbba157f3/xsdlib/src/main/java/com/sun/msv/datatype/regexp/InternalImpl.java has it different, though. I guess it would make sense to align the text with the reference either way?

Good point as this is matched to this minor variant at https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/rules/bsd-3-clause-sun_4.RULE

Now I am fine either way wrt. to spaces, that we have had this text with this spacing after the dashes roughly for 5 years https://github.com/nexB/scancode-toolkit/blob/166d565721ed1210f5cab3f7d04415654ee86869/src/licensedcode/data/licenses/bsd-3-clause-sun.LICENSE
And the current form (cleaned from leading ASCII decoration) for 3 years https://github.com/nexB/scancode-toolkit/blob/9677140cdc3a3e96fd3c424fcc7eb0cc19533b66/src/licensedcode/data/licenses/bsd-3-clause-sun.LICENSE )
while this has been in SPDX only for 5 months https://github.com/spdx/license-list-XML/commits/main/src/BSD-3-Clause-Sun.xml

PR welcome!
